src/main/java/org/springframework/data/elasticsearch/core/EntityMapper.java

```diff
@@ -32,12 +32,14 @@
  */
 public interface EntityMapper {
 
-	String mapToString(Object object) throws IOException;
+	default String mapToString(Object object) throws IOException {
```

Comment:
We should remove `EntityMapper` entirely.

Reply:
see above